Random_numbers/include/CGAL/Random.h

```diff
@@ -232,7 +232,7 @@ class Random {
 
 inline Random& get_default_random()
 {
-#if (defined( CGAL_TEST_SUITE ) || defined( CGAL_PRINT_SEED )) && !defined(CGAL_HEADER_ONLY)
+#if (defined( CGAL_TEST_SUITE ) || defined( CGAL_PRINT_SEED ))
```

Comment:
You can remove the pair of parenthesis, now:
```suggestion
#if defined( CGAL_TEST_SUITE ) || defined( CGAL_PRINT_SEED )
```